sql/src/main/java/org/apache/druid/sql/calcite/rel/DruidQuery.java

```diff
@@ -984,97 +1001,27 @@ public ScanQuery toScanQuery()
                            ? 0L
                            : (long) limitSpec.getLimit();
 
+    ScanQuery.Order order;
+    if (limitSpec == null || limitSpec.getColumns().size() == 0) {
+      order = ScanQuery.Order.NONE;
+    } else if (limitSpec.getColumns().get(0).getDirection() == OrderByColumnSpec.Direction.ASCENDING) {
```

Comment:
bit of a tangent, but why again did we add a new `ScanQuery.Order` instead of re-using `OrderByColumnSpec` stuff?

Reply:
I think we decided on adding a `ScanQuery.Order enum` because `OrderByColumnSpec` didn't really support a no-ordering case.  See https://github.com/apache/incubator-druid/pull/7133#discussion_r266170777